src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -23,10 +24,17 @@
 
 (defn- render-fn
   [item]
-  (let [network-details (rf/sub [:wallet/network-details])]
+  (let [network-details     (rf/sub [:wallet/network-details])
+        network-short-names (map network-utils/network->short-name
+                                 (:network-preferences-names item))
+        prefix              (when (< (count network-short-names) 3)
+                              (network-utils/short-names->network-preference-prefix network-short-names))
+        address             (str prefix (:address item))]
     [quo/account-item
-     {:on-press      #(on-press (:address item) network-details)
-      :account-props item}]))
+     {:on-press      #(on-press address network-details)
```

Comment:
not related directly to your code but it seems `on-press` is a function defined in this file. 
We should probably update it to something more meaningful that expresses the use case for it.
Can be handled separately ofc if it's too much to change in this pr 👍

Reply:
Done